src/Microsoft.Windows.CsWin32/Generator.cs

```diff
@@ -477,34 +492,35 @@ private IEnumerable<MemberDeclarationSyntax> NamespaceMembers
         {
             IEnumerable<IGrouping<string, MemberDeclarationSyntax>> members = this.committedCode.MembersByModule;
             IEnumerable<MemberDeclarationSyntax> result = Enumerable.Empty<MemberDeclarationSyntax>();
-            for (int i = 0; i < members.Count(); i++)
+            int i = 0;
+            foreach (IGrouping<string, MemberDeclarationSyntax> entry in members)
             {
-                IGrouping<string, MemberDeclarationSyntax> entry = members.ElementAt(i);
-                if (i == 0)
+                ClassDeclarationSyntax partialClass = DeclarePInvokeClass(entry.Key)
+                    .AddMembers(entry.ToArray())
+                    .WithLeadingTrivia(ParseLeadingTrivia(string.Format(CultureInfo.InvariantCulture, PartialPInvokeContentComment, entry.Key)));
+                if (i++ == 0)
```

Comment:
[nit] May be we could avoid combining two operations as it could confuse some readers, and may inadvertenly introduce bugs if this code is modified/extended later.

Reply:
I've pushed [a change](5632b2d840ebe5ee698379088f20bf4c333dd4de) to apply this.

Reply:
🚀